src/emulator/cpu.ts

```diff
@@ -1054,12 +1087,16 @@ export class Cpu {
                 this.bus.write(this.bus.read16((this.state.p + 0x01) & 0xffff), value & 0xff);
                 break;
 
+            case AddressingMode.imm16ind16:
```

Comment:
Sollte da nicht in Klammern `value & 0xffff` stehen und der Wert von außen kommen?

Reply:
Was meinst du? Verstehe leider deinen Kommentar nicht. ;)

Reply:
Hast recht, der Kommentar ist Quatsch, ich habe das `value` nicht gesehen und gedacht, dass Du hier auch das Laden des Arguments machts.